sdks/java/core/src/test/java/org/apache/beam/sdk/testing/TestPipelineTest.java

```diff
@@ -208,21 +208,21 @@ public String apply(final String input) {
       @Rule
       public final transient RuleChain chain = RuleChain.outerRule(exception).around(pipeline);
 
-      @Category(ValidatesRunner.class)
+      @Category(NeedsRunner.class)
```

Comment:
This one is also iffy, maybe? But really these tests are also violating the core tenet of tests which is straight-line readability. So I can't actually tell at a glance what kind of test they really are. So go ahead and leave this as you have it, I guess.

Reply:
Ack